wagtail/admin/templates/wagtailadmin/reports/listing/_list_page_types_usage.html

```diff
@@ -0,0 +1,93 @@
+{% load i18n wagtailadmin_tags wagtailcore_tags %}
+
+<table class="listing {% block table_classname %}{% endblock table_classname %}">
```

Comment:
It seems a bit odd to have to create a custom template for a report when we have most of the structure in python. Maybe that's something we can look at in future.

Reply:
I was following a structure similar to the [`base_page_report.html`](https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templates/wagtailadmin/reports/base_page_report.html), as it was more readable than putting everything into a single template.

Reply:
This wasn't necessarily directed at you - I was just thinking more from a framework perspective... This could be dynamic by default really, and you should only need to specify a template if you needed to override something.